fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/NumericArithmetic.java

```diff
@@ -1072,7 +1072,7 @@ public static Expression dlog1(DoubleLiteral first) {
         if (inputOutOfBound(first, 0.0d, Double.MAX_VALUE, false, true)) {
             return new NullLiteral(DoubleType.INSTANCE);
         }
-        return checkOutputBoundary(new DoubleLiteral(Math.log1p(first.getValue())));
+        return checkOutputBoundary(new DoubleLiteral(Math.log(first.getValue())));
```

Comment:
reuse `ln` and remove dlog1 function in FE

Reply:
fixed